demo/demo_director.py

```diff
@@ -742,8 +742,30 @@ def listen():
 
   server.register_function(clear_vehicle_targets, 'clear_vehicle_targets')
 
-  server.register_function(mitm_arbitrary_package_attack, 'mitm_arbitrary_package_attack')
-  server.register_function(undo_mitm_arbitrary_package_attack, 'undo_mitm_arbitrary_package_attack')
+  # Attack 1: Arbitrary Package, no keys
+  server.register_function(mitm_arbitrary_package_attack,
+      'mitm_arbitrary_package_attack')
+  server.register_function(undo_mitm_arbitrary_package_attack,
+      'undo_mitm_arbitrary_package_attack')
+
+  # Attack 2: Replay, no keys
+  server.register_function(prepare_replay_attack_nokeys,
```

Comment:
This is not a rollback attack, though. We're replaying old metadata which, even if it were accepted, would not move the client to a lower version of an image.